src/pudl/transform/eia861.py

```diff
@@ -1514,6 +1516,71 @@ def dynamic_pricing(tfr_dfs):
     return tfr_dfs
 
 
+def energy_efficiency(tfr_dfs):
+    """
+    Transform the EIA 861 Energy Efficiency table.
+
+    Args:
+        tfr_dfs (dict): A dictionary of transformed EIA 861 DataFrames, keyed by table
+            name. It will be mutated by this function.
+
+    Returns:
+        dict: A dictionary of transformed EIA 861 dataframes, keyed by table name.
+
+    """
+    idx_cols = [
+        'utility_id_eia',
+        'state',
+        'balancing_authority_code_eia',
+        'report_date',
+    ]
+
+    raw_ee = tfr_dfs["energy_efficiency_eia861"].copy()
+
+    # No duplicates to speak of but take measures to check just in case
+    _check_for_dupes(raw_ee, 'Energy Efficiency', idx_cols)
+
+    ###########################################################################
+    # Tidy Data:
+    ###########################################################################
+
+    logger.info("Tidying the EIA 861 Energy Efficiency table.")
+
+    # wide-to-tall by customer class (must be done before wide-to-tall by fuel class)
+    tidy_ee, ee_idx_cols = pudl.transform.eia861._tidy_class_dfs(
```

Comment:
Codacy found an issue: [Unused variable 'ee_idx_cols'](https://app.codacy.com/manual/zaneselvans/pudl/pullRequest?prid=6033440)

Reply:
I think you can assign the `ee_idx_cols` to be `_`, which effectively doesn't save it.